fix_includes.py

```diff
@@ -2142,10 +2142,27 @@ def FixFileLines(iwyu_record, file_lines, flags, fileinfo):
   # Add a sentinel decorated move-span, to make life easy, and sort.
   decorated_move_spans.append(((len(file_lines), len(file_lines)),
                                _EOF_KIND, '', []))
-  if flags.reorder:
-    decorated_move_spans.sort()
-  else:
-    decorated_move_spans.sort(key=lambda x: x[0:-2])
+
+  quoted_first = {
```

Comment:
It took me a while to decode what was going on here. I think it would make sense to break it out into a helper method. The C++ side has `GetLineSortOrdinal`: https://github.com/include-what-you-use/include-what-you-use/blob/57bfcc4b3f98b0c26273f9e2465069b062cdc9f8/iwyu_output.cc#L2043.

I don't think we need to do as much active parsing, but it might be nice to do something like:
```
def GetLineSortOrdinal(kind, quoted_includes_first):
    if not quoted_includes_first:
        return kind
    # ... otherwise build the translation table with a little comment to explain...
    sort_order = {...}
    return sort_order[kind]
```
(not literally that, but something nice like it)